### ISSUE TITLE
Linuxkit errors out when trying to run `make image-amd64`

### ISSUE THREAD

**Issue by c0dyhi11** [tinkerbell/hook#63]
I had to remove `-docker` from the following line in order to proceed

https://github.com/tinkerbell/hook/blob/6f97067fe0912c741eb9bd1798b9c62574e5ef78/Makefile#L25

Here is the error:
```
# make image-amd64
mkdir -p out
linuxkit build -docker -disable-content-trust -pull -format kernel+initrd -name hook-x86_64 -dir out hook.yaml
flag provided but not defined: -docker
USAGE: linuxkit build [options] <file>[.yml] | -

Options:
  -decompress-kernel
    	Decompress the Linux kernel (default false)
  -dir string
    	Directory for output files, default current directory
  -disable-content-trust
    	Skip image trust verification specified in trust section of config (default false)
  -format value
    	Formats to create [ aws docker dynamic-vhd gcp iso-bios iso-efi kernel+initrd kernel+iso kernel+squashfs qcow2-bios qcow2-efi raw-bios raw-efi rpi3 tar tar-kernel-initrd vhd vmdk ]
  -name string
    	Name to use for output files
  -o string
    	File to use for a single output, or '-' for stdout
  -pull
    	Always pull images
  -size string
    	Size for output image, if supported and fixed size (default "1024M")
make: *** [Makefile:25: image-amd64] Error 2
```

I was not able to get `nix-shell` working so I installed `linux-kit v0.8`
That may be the problem...

**Comment by deitch** [comment]
v0.8 is old...

**Comment by gianarb** [comment]
You are not using the right version.

**Comment by c0dyhi11** [comment]
https://github.com/linuxkit/linuxkit/releases/tag/v0.8

This is the latest release… Are binaries available elsewhere?

**Comment by deitch** [comment]
No unfortunately. I'd open an issue on the repo that the latest release is a year old. I'll see if I can cut one.

**Comment by c0dyhi11** [comment]
And using the most up to date linuxkit (Built from source this morning) you now get this error on that same command:
```
flag provided but not defined: -disable-content-trust
```

**Comment by deitch** [comment]
Oh yeah. notary v1 support was removed. It was a pain in the neck, didn't really work, no one wanted it, and notary v2 (under construction by the various SIGs and such) will be 100% non-backwards compatible.

**Comment by micahhausler** [comment]
This should be fixed now since #60 has merged, @c0dyhi11 can you confirm?

**Comment by c0dyhi11** [comment]
Yep! Thanks!
